### PR TITLE
Restructure the logs page

### PR DESCRIPTION
**Pull request by cartermp**
Refocused this page to be more end-user oriented. Also elaborated on the different log types.

### CODE REVIEW

File: content/en/docs/concepts/signals/logs.md [hunk 1]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
-OpenTelemetry's approach with logs is different. Because existing logging
-solutions are widespread in language and operational ecosystems, OpenTelemetry
-acts as a "bridge" between those logs, the tracing and metrics signals, and
-other OpenTelemetry components. In fact, the API for logs is called the "Logs
-Bridge API" for this reason.
+OpenTelemetry does not define a bespoke API or SDK to create logs. Instead,
+OpenTelemetry logs are simply the existing logs you already have from a logging
+framework or infrastructure component. Under the covers, OpenTelemetry SDKs and
+autoinstrumentation utilize several components to automatically correlate logs
+with [traces](/docs/signals/traces).
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:18]:
```suggestion
OpenTelemetry does not define a bespoke API or SDK to create logs. Instead,
OpenTelemetry logs are the existing logs you already have from a logging
framework or infrastructure component. OpenTelemetry SDKs and
autoinstrumentation utilize several components to automatically correlate logs
with [traces](/docs/signals/traces).
```

File: content/en/docs/concepts/signals/logs.md [hunk 2]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
 
-The [logs specification][] contains more details on this philosophy.
+OpenTelemetry's support for logs is designed to be maximally compatible with
+what you already have, providing capabilities to wrap those logs with additional
+context and a common toolkit to parse and manipulate logs into a common format
+across many different sources.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:23]:
```suggestion
OpenTelemetry's support for logs is designed to be fully compatible with
what you already have, providing capabilities to wrap those logs with additional
context and a common toolkit to parse and manipulate logs into a common format
across many different sources.
```

> what you already have,

Could we be more specific?

File: content/en/docs/concepts/signals/logs.md [hunk 3]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
-## Log Appender / Bridge
+The [OpenTelemetry Collector](/docs/collector) provides several tools to work
+with logs:
+
+- Several receivers which parse logs from specific, known sources of log data
+- The filelogreceiver, which reads logs from any file and provides features to
+  parse them from different formats or use a regular expression
+- Processors like the transformprocessor which lets you parse nested data,
+  flatten nested structures, add/remove/update values, and more
+- Exporters that let you emit log data in a non-OpenTelemetry format
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:35]:
```suggestion
- Several receivers which parse logs from specific, known sources of log data.
- The filelogreceiver, which reads logs from any file and provides features to
  parse them from different formats or use a regular expression.
- Processors like the transformprocessor which lets you parse nested data,
  flatten nested structures, add/remove/update values, and more.
- Exporters that let you emit log data in a non-OpenTelemetry format.
```

File: content/en/docs/concepts/signals/logs.md [hunk 4]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+
+Often times, the first step in adopting OpenTelemetry involves deploying a
+Collector as a general-purposes logging agent.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:38]:
```suggestion
The first step in adopting OpenTelemetry frequently involves deploying a
Collector as a general-purposes logging agent.
```

File: content/en/docs/concepts/signals/logs.md [hunk 5]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+
+### OpenTelemetry Logs for Applications
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:40]: All headings should be sentence cased.

File: content/en/docs/concepts/signals/logs.md [hunk 6]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+
+In applications, OpenTelemetry logs are created with any logging library or
+built-in logging capabilities. When you add autoinstrumentation or activate an
+SDK, OpenTelemetry will automatically correlate your existing logs with any
+active trace and span, wrapping the log body with their IDs. In other words,
+OpenTelemetry will automatically correlate your logs!
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:46]:
```suggestion
In applications, OpenTelemetry logs are created with any logging library or
built-in logging capabilities. When you add autoinstrumentation or activate an
SDK, OpenTelemetry will automatically correlate your existing logs with any
active trace and span, wrapping the log body with their IDs. In other words,
OpenTelemetry automatically correlates your logs and traces.
```

File: content/en/docs/concepts/signals/logs.md [hunk 7]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+the OpenTelemetry specification. For the individual language specific
+implementations of the Logs API & SDK, the status is as follows:
+
+{{% signal-support-table "logs" %}}
+
+## Structured, Unstructured, and Semistructured Logs
+
+OpenTelemetry does not distinguish between structured and unstructured logs.
+However, it is vitally important to understand the different kinds of formats
+that log data can manifest in.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:60]:
```suggestion
OpenTelemetry does not distinguish between structured and unstructured logs.
However, it is vitally important to understand the different kinds of formats
that log data can manifest in.
```

If OTel doesn't distinguish log types, why understanding the different kinds matter? I think this intro would benefit from a rewrite that focuses on why one should continue reading.

File: content/en/docs/concepts/signals/logs.md [hunk 8]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+example, and Extended Log Format (ELF) log mixes JSON with the
+whitespace-separated data in a CLF log.
+
+```text
+192.168.1.1 - johndoe [04/Aug/2024:12:34:56 -0400] "POST /api/v1/login HTTP/1.1" 200 1234 "http://example.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36" {"transactionId": "abcd-efgh-ijkl-mnop", "responseTime": 150, "requestBody": {"username": "johndoe"}, "responseHeaders": {"Content-Type": "application/json"}}
+```
+
+To make the most use of this log, it is recommended to parse bothe the JSON and
+the ELF-related pieces into a shared format to make analysis on an Observability
+backend easier.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:120]:
```suggestion
To make the most use of this log, parse both the JSON and
the ELF-related pieces into a shared format to make analysis on an observability
backend easier.
```

File: content/en/docs/concepts/signals/logs.md [hunk 9]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+consistent format, they are easier to parse, which makes them easier to
+preprocess in an OpenTelemetry Collector, correlate with other data, and
+ultimate analyze in an Observability backend. Use structured logs if you can.
+
+### Unstructured Logs
+
+Unstructured logs are logs that don't follow a consistent structure. They may be
+more human-readable, and are often used in development. However, it is not
+preferred to use unstructured logs to contain useful debugging data if you can
+help it.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:132]:
```suggestion
Unstructured logs are logs that don't follow a consistent structure. They may be
more human-readable, and are often used in development.
```

I think the recommendation is really a preference here, and not something always actionable. Plus, passive voice is a red flag. :)

File: content/en/docs/concepts/signals/logs.md [hunk 10]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+[ERROR] 2024-08-04 12:45:23 - Failed to connect to database. Exception: java.sql.SQLException: Timeout expired. Attempted reconnect 3 times. Server: db.example.com, Port: 5432
+
+System reboot initiated at 2024-08-04 03:00:00 by user: admin. Reason: Scheduled maintenance. Services stopped: web-server, database, cache. Estimated downtime: 15 minutes.
+
+DEBUG - 2024-08-04 09:30:15 - User johndoe performed action: file_upload. Filename: report_Q3_2024.pdf, Size: 2.3 MB, Duration: 5.2 seconds. Result: Success
+```
+
+To reiterate, structured logs are fine for debugging applications locally, but
+are very difficult to use at scale when observing large systems. It is
+recommended that you use structured logging if possible.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:146]:
```suggestion
Structured logs are fine for debugging applications locally, but
are very difficult to use at scale when observing large systems.
Use structured logging if possible.
```

File: content/en/docs/concepts/signals/logs.md [hunk 11]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+Example of a semistructured log:
+
+```text
+2024-08-04T12:45:23Z level=ERROR service=user-authentication userId=12345 action=login message="Failed login attempt" error="Invalid password" ipAddress=192.168.1.1 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36"
+```
+
+Although machine-readable, semistructured logs may require several different
+parsers to allow for analysis at scale.
+
+## OpenTelemetry Logging Componentry
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:163]: > Componentry

Can we use a different word here?

File: content/en/docs/concepts/signals/logs.md [hunk 12]
```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
...
+
+The following lists of concepts/components are what powers OpenTelemetry's
+logging support under the hood.
```
Comment (theletterf) [inline content/en/docs/concepts/signals/logs.md:166]:
```suggestion
The following lists of concepts and components power OpenTelemetry's
logging support.
```

**Review by theletterf — CHANGES_REQUESTED** [review]
First pass. Good work!

**Comment by cartermp** [comment]
@theletterf all addressed

**Review by theletterf — APPROVED** [review]

**Comment by chalin** [comment]
FYI, it seems that this PR introduced some spell-check issues, see:

- #4956

**Pull request merged**